UKHomeOffice/nmsw-ui
# NMSW-422 Update create-account/email-address text

# Ticket

NMSW-422

----

## AC

Chnage content in the inset text on the create-account/email-address page

----

## To test

- Run app
- Go to localhost:3000
- Click on create an account
- The text on the email page should match that of the prototype

----

## Notes
